src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -143,16 +141,14 @@
                         </assembly>
                         <assembly ref="parameter-guideline" max-occurs="unbounded">
                                 <use-name>guideline</use-name>
-                                <!-- CHANGED from "guidance" to "guidelines" -->
                                 <group-as name="guidelines" in-json="ARRAY"/>
                         </assembly>
                         <choice>
                                 <field ref="parameter-value" max-occurs="unbounded">
-                                        <!-- CHANGED cardinality to allow for multiple values -->
                                         <use-name>value</use-name>
                                         <group-as name="values" in-json="ARRAY"/>
                                         <remarks>
-                                                <p>A set of values provided in a catalog can be redefined at any higher layer of OSCAL (e.g., Profile).</p>
+                                                <p>A set of values provided in a catalog can be redefined at any higher layer model of OSCAL (e.g., Profile).</p>
```

Comment:
A further tweak based during the review meeting.

```suggestion
                                                <p>A set of values provided in a catalog can be redefined in <code>profile</code> or <code>system-security-plan</code> models.</p>
```

Reply:
Not gonna use the suggestion, just will commit locally and push up in a touched up commit.